test/spec/tpmnBidAdapter_spec.js

```diff
@@ -141,4 +159,44 @@ describe('tpmnAdapterTests', function() {
       expect(bidResponses).deep.equal([mockBidResult]);
     });
   });
+
+  describe('getUserSync', function() {
+    const KEY_ID = 'uuid';
+    const TMP_UUID = generateUUID().replace(/-/g, '');
+    it('getCookie mock Test', () => {
```

Comment:
these tests will all fail in prebid 7 where you won't have access to storage manager unless the pub has whitelisted. please make your tests pb7 compatible.

Reply:
thx. comment. fix 'getUserSyncs' test case code, And  add storageAllowed policy from pb7